dll/win32/msgina/lang/bg-BG.rc

```diff
@@ -172,12 +172,17 @@ BEGIN
     DEFPUSHBUTTON "OK", IDOK, 115, 179, 50, 14
 END
 
-IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 275, 146
+IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 275, 135
 STYLE NOT WS_VISIBLE | DS_CENTER | DS_MODALFRAME | DS_SHELLFONT | WS_BORDER | WS_CAPTION | WS_DLGFRAME | WS_POPUP
-CAPTION "Luna-XP-style Shut Down dialog"
+CAPTION "Изключване на РеактОС"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    DEFPUSHBUTTON "OK", IDOK, 80, 122, 55, 14, WS_GROUP
+    CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
+    CONTROL IDI_BAR, IDC_BAR, "Static", SS_BITMAP, 0, 44, 275, 4
+    PUSHBUTTON "Slee&p", IDC_BUTTON_SLEEP, 37, 63, 55, 31, BS_MULTILINE | WS_TABSTOP | WS_GROUP
```

Comment:
`WS_TABSTOP` is default value. Is `WS_GROUP` necessary?

Reply:
Not very sure here, maybe I need to leave only `WS_GROUP`. The idea here is to switch between power buttons with arrow keys, and tab between the group of power buttons and a Cancel button.

Reply:
OK.